### DISCUSSION TITLE
Webhook does not send base64 videos and documents

### DISCUSSION THREAD

**Discussion by fhonline** [salman0ansari/whatsapp-api-nodejs#365] [Q&A]
because I don't receive in the webhook, the base64 of videos and files like pdf, xls, doc... is there any configuration in the api that releases this?

**Comment by salman0ansari** [comment]
https://github.com/salman0ansari/whatsapp-api-nodejs/blob/2c9f0c267bab00943c1a52894413b3aef69cbb42/.env.example#L38
